projects/storefrontlib/cms-components/product/product-list/product-list-item/product-list-item.component.html

```diff
@@ -1,6 +1,12 @@
 <div class="row">
   <div class="col-12 col-md-4">
     <a
+      [attr.aria-label]="
+        !product.images?.hasOwnProperty('PRIMARY')
+          ? ('productDetails.noProductImage'
+            | cxTranslate: { product: product.name })
+          : ''
```

Comment:
`should:` please set `undefined` if this attribute is not need, as with empty string `aria-label` will still exist for an element